### PR TITLE
i18n(ko-KR): add `ecommerce.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description (required)

- add `ecommerce.mdx`

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Review by TheOtterlord — COMMENTED** [review]
Suggestions porting the changes from the other PR

File: src/content/docs/ko/guides/ecommerce.mdx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+      data-item-custom1-name="Frame color"
+      data-item-custom1-options="Brown|Silver[+10.00]|Gold[+20.00]"
+      data-item-custom2-name="Delivery instructions"
+      data-item-custom2-type="textarea"
+    >
+      Add to cart
+    </button>
+    ```
+
+4. `snipcart-checout` 클래스와 함께 Snipcart 체크아웃 버튼을 추가하여 카트를 열고 손님이 체크아웃 모달로 구매를 완료할 수 있도록 하세요.
```
[the remark this answers is not in the document]
Comment (jsparkdev) [reply src/content/docs/ko/guides/ecommerce.mdx:167]: thank you!

File: src/content/docs/ko/guides/ecommerce.mdx [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+      window.SnipcartSettings = {
+        publicApiKey: "YOUR_API_KEY",
+        loadStrategy: "on-user-interaction",
+      };
+
+      (function()...); // Snipcart 문서를 참조하세요.
+    </script>
+    ```
+
+2. [사용 가능한 Snipcart 설정](https://docs.snipcart.com/v3/setup/installation#setting2)을 사용하여 `window.SnipcartSettings`를 맞춤설정하여 장바구니의 동작과 모양을 제어하세요.
```
[the remark this answers is not in the document]
Comment (jsparkdev) [reply src/content/docs/ko/guides/ecommerce.mdx:132]: thank you!

**Review by yanthomasdev — APPROVED** [review]
LGTM! Good work 🙌

**Pull request merged**